source/extensions/transport_sockets/tls/ssl_socket.h

```diff
@@ -70,8 +70,6 @@ class SslSocket : public Network::TransportSocket,
   void onFailure() override;
   Network::TransportSocketCallbacks* transportSocketCallbacks() override { return callbacks_; }
 
-  SSL* rawSslForTest() const { return rawSsl(); }
```

Comment:
Huh, I guess this wasn't used?  Thanks for the cleanup.

Reply:
yes and it was also messing with coverage so I removed it